README.md

```diff
@@ -1426,6 +1426,28 @@ When everything is working as expected locally using `mp-test`, we can run
 
 </details>
 
+<details>
+  <summary><h3> How do I change the log level (SDK tools)?</h3></summary>
+
+  All SDK tools read the LOG_LEVEL environment variable to set the log level of their console output. For example, to set log level
+  to debug to see a verbose output of the any of the CLI tools we can set the LOG_LEVEL variable:
+
+  For Linux and macOS
+  ```shell
+  LOG_LEVEL=debug mp-build
+  ```
+  For Windows
+  ```
+  set LOG_LEVEL=debug
+  mp-build
+  ```
+  For Windows, set the log level back to `info` after debugging.
+
+  All logs can be seen in the command line, but they are also saved in  `logs/build.log` with `debug` log level. CLI tools generate logs
+  for `debug`, `warn`, `info`, and `error` level.
```

Comment:
This is confusing, because I _think_ it's just referring to the specific example of `mp-build` at `debug` level, but it reads more generally. I think we can probably remove most of it and just say what the log level options are.